Dockerfile

```diff
@@ -8,10 +8,14 @@ ENV ONE_DIR=0
 # Packages
 RUN apt-get update -q --fix-missing && \
   apt-get -y upgrade && \
+  apt-get -y install postfix
+RUN DEBIAN_FRONTEND=noninteractive apt-get update -q --fix-missing && \
```

Comment:
Iirc this was the only way it worked. Putting it in the already defined one doesn't complete postfix install